src/onegov/org/views/resource.py

```diff
@@ -595,20 +596,21 @@ def view_resource(
 def handle_delete_resource(self: Resource, request: 'OrgRequest') -> None:
 
     request.assert_valid_csrf_token()
-
-    # FIXME: should we move this attribute to the base class?
-    #        it could just be statically True or False by default
-    if not self.deletable:  # type:ignore[attr-defined]
-        raise exc.HTTPMethodNotAllowed()
-
     tickets = TicketCollection(request.session)
 
     def handle_reservation_tickets(reservation: 'BaseReservation') -> None:
         ticket = tickets.by_handler_id(reservation.token.hex)
         if ticket:
             assert request.current_user is not None
+
             close_ticket(ticket, request.current_user, request)
-            ticket.create_snapshot(request)
+            if not ticket.snapshot:
+                ticket.create_snapshot(request)
```

Comment:
Shouldn't we always create a snapshot? Otherwise the snapshot will start to diverge from the actual resource.